dev/com.ibm.ws.kernel.boot/resources/com/ibm/ws/kernel/boot/resources/LauncherMessages.nlsprops

```diff
@@ -151,7 +151,7 @@ error.creatingNewServer=CWWKE0030E: An exception occurred while creating server
 error.creatingNewServer.explanation=The specified server could not be created.
 error.creatingNewServer.useraction=Check and resolve the exception, then try again to create the server.
 
-error.noExistingServer=CWWKE0031E: Specified server {0} does not exist; use the --create option to create a new server. serverPath: {1}
+error.noExistingServer=CWWKE0031E: Specified server {0} does not exist; use the create action to create a new server. serverPath: {1}
```

Comment:
Specified server {0} does not exist
-->
Specified {0} server  does not exist

Reply:
fixed